static/bidder-info/thetradedesk.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "https://direct.adsrvr.org/bid/bidder/{{.SupplyId}}"
+maintainer:
+  email: "Prebid-Maintainers@thetradedesk.com"
+gvlVendorID: 21
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  redirect:
+    url: https://match.adsrvr.org?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
```

Comment:
This URL is giving me a 404. If a URL is provided it needs to work out-of-the-box. If you support user sync but require some manual steps you can indicate that you support redirect here along with a comment indicating that the host must reach out to you to configure user sync.

Reply:
It needs a configured and valid partner id in order to work. E.g:
https://match.adsrvr.org/track/rid?ttd_pid=67jqghy&fmt=json